tests/scripts/pulp_deb/test_deb_remote.sh

```diff
@@ -72,5 +72,10 @@ expect_succ pulp deb remote update --name "${ENTITIES_NAME}" --architecture ""
 expect_succ pulp deb remote show --name "${ENTITIES_NAME}"
 assert "$(echo "$OUTPUT" | jq -r .architectures)" == "null"
 
+#Try to add a gpgkey
+expect_succ pulp deb remote update --name "${ENTITIES_NAME}" --gpgkey "mygpgkey"
```

Comment:
I take it there is not validation yet that this looks even remotely like a key?
Still, there is a gpg key for testing purposes around. We use for testing.
It comes from here: https://github.com/pulp/pulp-fixtures/tree/master/common
And the private key is already downloaded and installed via pytest fixtures.
Can you try using `$(gpg --export-key ...)` to obtain it's public part?

Reply:
I will try :)

For what I saw, there is no validation done by the software for the format of the key.

But if the key is incorrect, it will fail during the repository sync operation

Reply:
sure. I was just thinking that validation would not be a bad idea, and we probably don't want to set this test up for failure. If it turns out to be not so trivial, we can think about postponing it again.
OTOH It would be nice to perform the actual sync test with that key. I think the fixtures are signed accordingly.